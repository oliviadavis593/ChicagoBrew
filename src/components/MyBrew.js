import React, { Component } from 'react';
import Navigation from '../components/Navigation';
import BrewContext from '../BrewContext';

class MyBrew extends Component {

    static contextType = BrewContext; 

    render() {
        return(
            <section>
                <div>
                    <Navigation />
                </div>
                <div className='brew-list'>
                    
                </div>
            </section>
        )
    }
}

export default MyBrew; 