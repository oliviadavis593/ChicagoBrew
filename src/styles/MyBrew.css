.cbt-mybrew-list {
   margin-right: 40px; 
}

.cbt-mybrew-list li {
    list-style-type: none;
      text-decoration: none;   
  }

.cbt-mybrew-list a {
    text-decoration: none; 
}


.cbt-mybrew-list button {
    width: 200px; 
}

.cbt-delete__container button {
    width: 120px;
    height: 35px; 
    font-family: 'Roboto Condensed', sans-serif;
    border: 4px solid #fff; 
    background-color: #000; 
    color: #fff;  
    font-size: 15px; 
}

.cbt-delete__container:hover button {
    color: #ffdfd3;
    transition: 0.7s;  
}