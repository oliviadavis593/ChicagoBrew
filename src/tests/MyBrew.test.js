import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import MyBrew from '../components/MyBrew';

const data = { params: {id: 2, name: 'test', }};

describe('MyBrew Component', () => {
    it('renders without crashing', () => {
        const div = document.createElement('div')
        ReactDOM.render(<BrowserRouter><MyBrew match={{...data}} /></BrowserRouter>, div)
        ReactDOM.unmountComponentAtNode(div)
    })

    it('renders the UI as expected', () => {
        const tree = renderer 
            .create(<BrowserRouter><MyBrew match={ {...data } }/></BrowserRouter>)
            .toJSON()
        expect(tree).toMatchSnapshot();
    })
})