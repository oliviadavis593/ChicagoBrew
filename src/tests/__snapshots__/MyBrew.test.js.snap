// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyBrew Component renders the UI as expected 1`] = `
<div>
  <div
    className="navigation"
  >
    <div
      className="cbt-navigation"
    >
      <div
        className="cbt-logo"
      >
        <a
          aria-current="page"
          className="active"
          href="/"
          onClick={[Function]}
          style={Object {}}
        >
          <img
            alt="cb-logo"
            className="cb-logo"
            src="chicagobrew-logo.png"
          />
        </a>
      </div>
      <nav
        className="nav"
      >
        <a
          aria-current={null}
          className="cbt-link__home"
          href="/dashboard"
          onClick={[Function]}
        >
          Home
        </a>
        <a
          aria-current={null}
          className="cbt-link__mybrew"
          href="/mybrews"
          onClick={[Function]}
        >
          My Brew
        </a>
      </nav>
    </div>
  </div>
  <ul
    className="cbt-mybrew-list"
  />
</div>
`;
